src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,17 +1,41 @@
 import React from 'react';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as filterActions } from '../../features/filter';
+import { Status } from '../../types/Status';
+
+const statuses = [Status.ALL, Status.ACTIVE, Status.COMPLETED];
```

Comment:
1. constants are named with uppercase
2. constants should be placed in separated file, for example `src -> constants -> statuses.ts` or even in your types/status.ts

```suggestion
const Statuses = [Status.ALL, Status.ACTIVE, Status.COMPLETED];
```